src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/InterArrival.java

```diff
@@ -15,6 +15,7 @@
  */
 package org.jitsi.impl.neomedia.rtp.remotebitrateestimator;
 
+import org.jitsi.impl.neomedia.rtp.TimestampUtils;
```

Comment:
open IntelliJ IDEA Settings → Code Style → Java, click Manage, and import that XML file by simply clicking Import.